handbook/bizops/index.md

```diff
@@ -57,6 +57,22 @@ We collect data from the following:
 
 We have [written policies about how we handle customer information](customer_data_policy.md). 
 
+## Debugging pings
+
+Use this section to debug ping abnormalities (from top to bottom).
```

Comment:
I'm not sure if the order someone should follow would always be "top to bottom". My hunch is that different signals in ping failures could lead us to want to test from one end first.

In general, it'd be nice to refer back to the "life of a ping" doc (or perhaps we move it to the handbook, or we put this section in the docs? I'm not totally convinced either way...). Then we can use this list as a guide for "how to debug each step" rather than a prescriptive "here's the order to debug in". 

What do you think?

Reply:
Also, this probably makes sense to move to a new file — this page is getting pretty dense :). Happy to help with this if that'd be helpful!

Reply:
@dadlerj agree with those suggestions. I've been thinking about changing the architecture of the page; I'll address that in a separate PR 👍